python/GafferSceneUI/EncapsulateUI.py

```diff
@@ -61,11 +61,11 @@
 
 	> Note : Encapsulation currently has some limitations
 	>
-	> - Motion blur options are taken from the globals at the
-	>   point of Encapsulation, not the downstream globals
-	>   at the point of rendering.
 	> - Motion blur attributes are not inherited - only
-	>   attributes within the encapsulate hierarchy are
+	>   attributes within the encapsulated hierarchy are
+	>   considered.
+	> - The `usd:purpose` attribute is not inherited - only
+	>   attributes withing the encapsulated hierarchy are
```

Comment:
`withing` -> `within`

Reply:
Thanks for spotting that - squashed a fix into https://github.com/GafferHQ/gaffer/pull/5474/commits/8f98b8e4dcab00f15f98a8d24738d30637cb2513.